packages/firebase_messaging/android/src/main/java/io/flutter/plugins/firebasemessaging/FlutterFirebaseMessagingService.java

```diff
@@ -42,4 +128,148 @@ public void onNewToken(String token) {
     intent.putExtra(EXTRA_TOKEN, token);
     LocalBroadcastManager.getInstance(this).sendBroadcast(intent);
   }
+
+  public static void startBackgroundIsolate(Context context, long callbackHandle) {
+    FlutterMain.ensureInitializationComplete(context, null);
+    String mAppBundlePath = FlutterMain.findAppBundlePath(context);
```

Comment:
I wouldn't use `m` here since it isn't a member.

Reply:
In the embedding I've migrated us away from hungarian notation. @collinjackson do you think we could migrate plugins away from it, too?

Reply:
I'll go with `appBundlePath`, I'll change if we decide otherwise.